### ISSUE TITLE
Proposal feedback

### ISSUE THREAD

**Issue by kaytwo** [MarcoBofa/final-project-MarcoB#1]
A lot of questions based on your proposal:

* What APIs will you be using? Are they free?
* Are you tracking what someone says they own and simulating trades, or actually enacting trades? If the latter, using what API/platform?
* How is this a unique project, compared to every web trading platform that already exists? What's the unique problem your app is solving that isn't already solved by other apps?
* A lot of this ChatGPT-output-looking text is very flashy but very under-defined/vague. You are free to use it to set your own project's expectations, but I am going to hold you to actually accomplishing what it says you're going to accomplish.
* Along with the previous point, this proposal sounds like it would take a 4 person group two semesters, not a one person group a third of a semester. Before the next checkpoint you should probably focus on a set of high impact (and preferably unique/novel) features that you want to implement.

**Comment by MarcoBofa** [comment]
Thank you for the detailed feedback,

I'm sorry if the pitch gave the impression of me trying to show off; that wasn't my intention. I have a somewhat vague idea of what I should or shouldn't aim for. Since this was my first pitch, I included every idea that came to mind, even though I know it's impossible (for me) to achieve all of that (unfortunately :c). 

For the APIs, I'm aware of a few that offer free (or nearly real-time) data for a vast number of stocks. My idea is for users to input their trades for simulation purposes, not actual trades. This, indeed, is one of the more ambitious (and “droppable") aspects of the project. The platform, in itself, isn't unique. However, I chose this approach because many platforms that offer alerts, DCF calculators, CAGR calculators, and portfolio management either charge users or only provide a subset of these tools. I fully understand its lack of uniqueness, but these were the ideas that came to mind.

For what concerns the ChatGPT-output-looking text, those were actually things that I wrote, just rivisited by GPT to correct my bad English knowledge.

So to sum up my actual idea was, to list all the features I'd like the app to include. By the next checkpoint, I plan to pare down the list, discarding any features I realize might be too challenging to implement. 
And another reason why I’ve choosen this, even if not so unique, it's because the app could be something that would be really useful for me and not just a random project.

So if the lack of uniqueness it acceptable I would like to continue with that otherwise I will (try to) find another idea.

Sorry again for the super arrogant pitch,

Thanks,

Marco.

**Comment by kaytwo** [comment]
I am 100% fine with ChatGPT generated text (for idea-fleshing-out or just
general English polishing), I think it just leads to a "marketing-speak"
type tone that doesn't exactly come across as arrogant but more "promise
everything." We are all learning how to use it (and live with it - even if
we aren't using it, lots of people will be suspicious as I was here),
please don't take my initial response as thinking it was arrogant or overly
inappropriate. I just saw it as "wow he really wants to do a lot" :).
Coming up with a lot of features and then paring it down is a fine idea.

For the project I like that you are excited about adding a lot of these
features, and adding features is definitely important, but the aspect I
really think is worth focusing on is making a _unique_ app: the question
I'd want you to answer for your own benefit is the "How is this a unique
project" component. Your final project will stand out from the crowd and be
a better conversation-starter in an interview if it is solving a unique
problem. Feel free to post on Piazza if you come up with further brainstorm
ideas.
